### PR TITLE
updated slack workflow

### PR DESCRIPTION
**Pull request by warunicorn19**
Signed-off-by: Aditya Chaterjee <speak2adi@gmail.com>

**Description**
Layer5 slack workflow fix
This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d44e9a418b7d510b2a5e339edd6d652b20787b69 at: https://611d6ba97d046000dbd23b21--layer5ng.netlify.app

**Review by sayantan1413 — APPROVED** [review]

File: .github/workflows/slack.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     steps:
     - name: Get current star count
       run: |
-        echo ::set-env name=STARS::$(curl --silent 'https://api.github.com/repos/layer5io/layer5' -H 'Accept: application/vnd.github.preview' | jq '.watchers_count')
+        echo "STARS=$(curl --silent 'https://api.github.com/repos/layer5io/layer5' -H 'Accept: application/vnd.github.preview' | jq '.stargazers_count')" >> $GITHUB_ENV
```
Comment (Nikhil-Ladha) [inline .github/workflows/slack.yml:10]: Shouldn't line: 16 be updated as we are saving the value to `GITHUB_ENV` now?

Comment (warunicorn19) [reply .github/workflows/slack.yml:10]: @Nikhil-Ladha i dont think so 🤔, this seems to be working for the other repositories.

Comment (leecalcote) [reply .github/workflows/slack.yml:10]: Multiple possible right answers, I suspect. Consistency is generally a good thing for us.

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**